$MirOS: ports/games/frozen-bubble/patches/patch-c_stuff_lib_fb_stuff_pm,v 1.1 2006/11/04 04:12:03 tg Exp $
--- c_stuff/lib/fb_stuff.pm.orig	Sat Nov 18 19:04:12 2006
+++ c_stuff/lib/fb_stuff.pm	Sat Dec 23 01:24:38 2006
@@ -24,7 +24,6 @@
 package fb_stuff;
 
 use fb_c_stuff;
-use Locale::gettext;
 use vars qw(@ISA @EXPORT $FPATH $FLPATH $FBLEVELS $colourblind %POS_1P %POS_2P %POS_MP $BUBBLE_SIZE $ROW_SIZE
             $PI $version);
 @ISA = qw(Exporter);
@@ -304,5 +303,5 @@ sub stringchars {
 
 sub t {
     my ($fmt, @args) = @_;
-    return sprintf(gettext($fmt), @args);
+    return sprintf($fmt, @args);
 }
